core/system/src/privcall.c

```diff
@@ -15,7 +15,18 @@
  * limitations under the License.
  */
 #include "api/inc/privcall_exports.h"
+#include <uvisor.h>
```

Comment:
You can add it there (vmpu.h). I agree it is better practice, even if we plan to make that inclusion redundant in the future.